spec/cli.spec.js

```diff
@@ -134,11 +134,11 @@ describe("cordova cli", function () {
 
     describe("create", function () {
         beforeEach(function () {
-            spyOn(cordova.raw, "create").andReturn(Q());
-            spyOn(cordova_lib, "CordovaError");
+            spyOn(cordova.raw, "create").and.returnValue(Q());
+            // spyOn(cordova_lib, "CordovaError");
```

Comment:
yes @stevengill ... it gives this error
```     Error: <spyOn> : CordovaError is not declared writable or has no setter
    Usage: spyOn(<object>, <methodName>) ```
Is this line still needed? I tried using this to fix it, but that did not work.
``` Object.defineProperty(cordova_lib, "CordovaError", { writable: true }); ```

I've seen some people fix it like this too, but not sure if this is correct.

```   var descriptor = Object.getOwnPropertyDescriptor(obj, methodName);
 if (!(descriptor.writable || descriptor.set)) {
throw new Error(methodName + ' is not declared writable or has no setter');
} ```